### PR TITLE
refactor: add lightweight style resources for TabBar

### PR DESCRIPTION
**Pull request by Marc-Antoine-Soucy**
GitHub Issue (If applicable): #
https://github.com/unoplatform/uno.toolkit.ui-private/issues/3

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] [Docs](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc) have been added/updated
- [ ] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by Marc-Antoine-Soucy** [comment]
waiting for this PR https://github.com/unoplatform/uno.toolkit.ui/pull/665 to pass before putting this one as ready for review

File: src/library/Uno.Toolkit.WinUI.Markup/Theme/TabBar.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{
+	public static class TabBar
+	{
+		public static class Resources
+		{
+			public static class Default
+			{
+				public static class Item
+				{
+					public static class Background
```
Comment (kazo0) [inline src/library/Uno.Toolkit.WinUI.Markup/Theme/TabBar.cs:17]: We should be creating just a `TabBarItem` class here that is outside of the `TabBar` class

File: src/library/Uno.Toolkit.WinUI.Markup/Theme/Card.cs
Comment (kazo0) [inline src/library/Uno.Toolkit.WinUI.Markup/Theme/Card.cs:1]: Duplicate of this PR? https://github.com/unoplatform/uno.toolkit.ui/pull/665

File: src/library/Uno.Toolkit.WinUI.Markup/ResourceValue.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+		public bool IsThemeResource { get; }
+
+		public static implicit operator Action<IDependencyPropertyBuilder<T>>(ResourceValue<T> resource) =>
+			resource.IsThemeResource ?
+				ThemeResource.Get<T>(resource.Key)
+				: StaticResource.Get<T>(resource.Key);
+
+	}
+}
```
[the remark this answers is not in the document]
Comment (Marc-Antoine-Soucy) [reply src/library/Uno.Toolkit.WinUI.Markup/ResourceValue.cs:24]: Do I need to install a package to access these classes, or is it already supposed to be in uno,toolkit?

File: src/library/Uno.Toolkit.WinUI.Markup/Theme/CardContentControl.cs
[the remark this answers is not in the document]
Comment (Marc-Antoine-Soucy) [reply src/library/Uno.Toolkit.WinUI.Markup/Theme/CardContentControl.cs:1]: This is not meant to include the cardContentControl, I just made this branch out of the other branch to have access to the classes created in the other pr, but this is no longer necessary if we just update the markeup nuget

**Comment by agneszitte** [comment]
Let's revisit if this PR is still needed after @Xiaoy312 current work with a script

**Pull request closed without merging**